pages/quick-start/new-project.md

```diff
@@ -14,6 +14,11 @@ The following npx command will create a new directory on your machine with the `
 ```bash
 npx @react-native-community/cli init --template=@react-native-firebase/template <name>
 ```
+> If the `cli init` script fails while performing the `pod install` phase (fails with `could not find spec` or similar), try the following step:
```

Comment:
Please remove the `>` characters, as it will cause the links to within a blockquote, with a nested code example.

Reply:
Done.
However I supposed this whole block to be displayed like a notion, not just regular text.
Anyway it looks ok after removal.